.github/workflows/mirror-helm-charts.yaml

```diff
@@ -0,0 +1,52 @@
+name: Update Addons
+on:
+  schedule:
+    # everyday at midnight.
+    - cron: '0 0 * * *'
+  workflow_dispatch: {}
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - name: Compile buildtools
+        run: |
+          make buildtools
+      - name: Upload buildtools artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: buildtools
+          path: output/bin/buildtools
+  update-dependencies:
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    strategy:
+      fail-fast: false
+      matrix:
+        addon:
+          - openebs
+          - registry
+          - seaweedfs
+          - velero
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v4
+      - name: Download buildtools artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: buildtools
+          path: ./output/artifacts
+      - name: Update ${{ matrix.addon }}
+        id: update
+        env:
+          DESTINATION: oci://ttl.sh/embedded-cluster-charts
```

Comment:
Could you set this as the upstream for the various charts + remove the additional registries, so that the integration tests can take a look at things?

Reply:
Yeah, that would be the next step. I would rather get this in and then work on that if that is OK to you.